runtime/lua/vim/lsp/client.lua

```diff
@@ -678,6 +671,24 @@ function Client:_request(method, params, handler, bufnr)
       params = params,
       version = version,
     }
+
+    if log.trace() then
+      log.trace(self.name, 'handler', method, { err = err, result = result, ctx = context })
+    end
+
+    -- Per LSP, don't show ContentModified error to the user.
+    if err and err.code ~= lsp.protocol.ErrorCodes.ContentModified then
+      -- (#25464) The haskell-language-server supports resolve only for a subset
```

Comment:
There is no ambiguity when mentioning `#xx` items, so don't need parens
```suggestion
      -- #25464 The haskell-language-server supports resolve only for a subset
```

Reply:
Not so about ambiguity, and more literal reading. I guess I can do `#25466: The ...` instead.